packages/shared-components/src/components/select/Combobox.tsx

```diff
@@ -58,12 +58,29 @@ const Combobox = forwardRef<HTMLInputElement, Props>(
       }
     };
 
+    const getSelectedOptions = () => {
+      if (value) {
+        if (typeof value === 'string') {
```

Comment:
For å slippe denne biten så kan vi kanskje transformere prefill-dataene ved populering slik at de ser ut slik vi forventer de? 

Altså, det objektet som sendes inn til `super.setValue` i linken nedenfor blir slik vi forventer at det ser ut i submission:
https://github.com/navikt/skjemabygging-formio/blob/master/packages/shared-components/src/formio/components/core/address/Address.tsx#L43